database-portfolio/src/main/webapp/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body onload="loadCommentSection(5)">
+    <div id="content">
+      <h1>Billy Porter</h1>
+      <p>Hi! My name is Billy Porter. I am a rising junior at the University of Notre Dame and am a
+          computer science major interested in deep learning and artificial intelligence. I am an avid
+          soccer player and scubadiver, and I enjoy personal fitness.
+      </p>
+      <img src="./images/SpiceAlley.JPG" alt="Image in Spice Alley" width="300" height="400">
+    </div>
+    <div id="post-content">
+        <h3>Set Maximum Number of Comments</h3>
+        <label>Max (between 1 and 10, default is 5): </label>
+        <input type="number" id="limit" name="limit" min="1" max="10">
+        <button onclick="loadCommentSection(document.getElementById('limit').value)">Refresh</button>
+        <h3>Comments: <h3>
+        <div class="comment-section">
+            <form action ="/comment-page" method="POST">
+                <label>Username</label>
+                <input type="text" name="user-comment">
+                <br><br>
```

Comment:
optional: I would personally use CSS here to add spacing in between the input boxes, but your choice.

Reply:
Definitely. I'm going to opt-out for now due to time and worry about stylistic parts after implementing in Angular